.github/workflows/test.yml

```diff
@@ -0,0 +1,132 @@
+name: Timber Tests
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    types: [opened, synchronize, ready_for_review]
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mariadb:latest
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: true
+          MYSQL_ROOT_PASSWORD:
+          MYSQL_DATABASE: wordpress_test
+        ports:
+          - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    continue-on-error: ${{ matrix.experimental == true }}
+    strategy:
+
+      fail-fast: false
+      matrix:
+        php: ['7.4', '7.3', '7.2'] # , '7.1', '7.0'
+        wp: ['5.3.6', 'latest']
+        multisite: ['0']
+        dependency-version: [prefer-stable] # prefer-lowest
+        webp: [false]
+        coverage: [false]
+        include:
+          # Multisite
+          - php: '7.4'
+            wp: 'latest'
+            dependency-version: 'prefer-stable'
+            multisite: '1'
+            coverage: false
+          # WP Trunk
+          - php: '7.4'
+            wp: 'trunk'
+            dependency-version: 'prefer-stable'
+            multisite: '0'
+            coverage: false
+            experimental: true
+          # Coverage
+          - php: '7.4'
+            wp: 'latest'
+            dependency-version: 'prefer-stable'
+            multisite: '0'
+            coverage: true
+
+    name: PHP ${{ matrix.php }} ${{ matrix.coverage && ' (with coverage)' || '' }} ${{ matrix.extensions && ' (with ' || '' }} ${{ matrix.webp && ' (webp)' || '' }} | WP ${{ matrix.wp }}${{ matrix.multisite == 1 && ' (MS)' || '' }} | ${{ matrix.dependency-version }}
```

Comment:
`matrix.extensions` is not defined.

Reply:
Old stuff forgotten. I removed it. But this still might be useful for the future. 

Some tests are properly set up (webp, gd/imagick)